src/app/child-dev-project/children/children.service.ts

```diff
@@ -79,70 +76,46 @@ export class ChildrenService {
         by_child: {
           map: `(doc) => {
             if (!doc._id.startsWith("${ChildSchoolRelation.ENTITY_TYPE}")) return;
-            emit(doc.childId);
-            }`,
+            emit([doc.childId, new Date(doc.start || '3000-01-01').getTime()]);
+          }`,
         },
         by_school: {
-          map: `(doc) => {
-            if ( (!doc._id.startsWith("${ChildSchoolRelation.ENTITY_TYPE}")) ||
-                (doc.start && isAfterToday(new Date(doc.start))) ||
-                (doc.end && !isAfterToday(new Date(doc.end))) ) {
-              return;
-            }
-            emit(doc.schoolId);
-            function isAfterToday(date) {
-              const tomorrow = new Date();
-              tomorrow.setHours(0, 0, 0, 0);
-              tomorrow.setDate(tomorrow.getDate() + 1);
-              return date >= tomorrow;
-            }
-            }`,
-        },
-        by_date: {
           map: `(doc) => {
             if (!doc._id.startsWith("${ChildSchoolRelation.ENTITY_TYPE}")) return;
-            let timestamp = (new Date(doc.start || '3000-01-01')).getTime().toString().padStart(14, "0");
-            emit(doc.childId + '_' + timestamp);
+            emit([doc.schoolId]);
```

Comment:
why not include the date here as well, so that by child/school can be used in the same way?

Reply:
There is no use case for it yet (the tables filter anyway again based on the first column).